nullaway/src/main/java/com/uber/nullaway/AbstractConfig.java

```diff
@@ -77,29 +77,25 @@ public abstract class AbstractConfig implements Config {
 
   protected boolean handleTestAssertionLibraries;
 
-  protected Set<String> optionalClassPaths;
+  protected ImmutableSet<String> optionalClassPaths;
 
   protected boolean assertsEnabled;
 
-  /**
-   * if true, {@link #fromAnnotatedPackage(Symbol.ClassSymbol)} will return false for any class
-   * annotated with {@link javax.annotation.Generated}
-   */
```

Comment:
No other such protected field has docs and besides these are stale (`#fromAnnotatedPackage` for this class no longer exists, it was never only that annotation, and after #600 this isn't even only [simple-name] `@Generated`), so...